### PR TITLE
Enable access logging on the SPA proxy

### PR DESCRIPTION
**Pull request by kylelaker-ccpo**

### CODE REVIEW

**Review by juliusfitzhugh-ccpo — APPROVED** [review]
Looks good to me. Now we are covered on the frontend and backend.

File: cdk/lib/apigateway-proxy.ts [hunk 1]
```diff
@@ -41,6 +44,28 @@ export class ApiGatewayProxy extends cdk.Construct {
...
+            // This field is in the Apache Common Log Format
+            requestTime: apigw.AccessLogField.contextRequestTime(),
+            httpMethod: apigw.AccessLogField.contextHttpMethod(),
+            protocol: apigw.AccessLogField.contextProtocol(),
+            responseLength: apigw.AccessLogField.contextResponseLength(),
+            status: apigw.AccessLogField.contextStatus(),
+            resourcePath: apigw.AccessLogField.contextResourcePath(),
+            path: apigw.AccessLogField.contextPath(),
+            ip: apigw.AccessLogField.contextIdentitySourceIp(),
+            userAgent: apigw.AccessLogField.contextIdentityUserAgent(),
```
Comment (juliusfitzhugh-ccpo) [inline cdk/lib/apigateway-proxy.ts:62]: Did you leave `user` out on purpose?

File: cdk/lib/apigateway-proxy.ts [hunk 2]
```diff
@@ -41,6 +44,28 @@ export class ApiGatewayProxy extends cdk.Construct {
...
+            // The Request ID for the integration (Lambda)
+            integrationRequestId: "$context.integration.requestId",
+            integrationLatency:
+              apigw.AccessLogField.contextIntegrationLatency(),
```
[the remark this answers is not in the document]
Comment (kylelaker-ccpo) [reply cdk/lib/apigateway-proxy.ts:66]: Nah, this is a formatting thing. This repo has tighter column requirements

File: cdk/lib/apigateway-proxy.ts (hunk 1 above)
Comment (juliusfitzhugh-ccpo) [reply cdk/lib/apigateway-proxy.ts:62]: Oh okay, yeah that make sense.

**Pull request merged**